Makefile

```diff
@@ -122,3 +122,6 @@ docker-build-minikube: docker-build-local
 	minikube image load ghcr.io/wundergraph/cosmo/keycloak:latest & \
 	minikube image load ghcr.io/wundergraph/cosmo/cdn:latest
 	minikube cache reload
+
+run-local-demo:
```

Comment:
Can we be more specifc? e.g. `run-subgraphs-local` ?

Reply:
Renamed